cli/src/commands/rebase.rs

```diff
@@ -193,10 +219,14 @@ Please use `jj rebase -d 'all:x|y'` instead of `jj rebase --allow-large-revsets
         simplify_ancestor_merge: false,
     };
     let mut workspace_command = command.workspace_helper(ui)?;
-    let new_parents = workspace_command
-        .resolve_some_revsets_default_single(&args.destination)?
-        .into_iter()
-        .collect_vec();
+    let new_parents = if args.destination.is_empty() {
+        vec![]
```

Comment:
This seems strange. Is this a placeholder value that to indicate something special? Can we use `Option<>` to model that instead?